src/accelerate/accelerator.py

```diff
@@ -2300,8 +2301,12 @@ def clip_grad_norm_(self, parameters, max_norm, norm_type=2):
                     xm.all_reduce("sum", gradients, scale=1.0 / self.num_processes)
                     # Set is_xla_gradients_synced to True to avoid all-reduce twice in the AcceleratedOptimizer step.
                     acc_opt.gradient_state.is_xla_gradients_synced = True
-        self.unscale_gradients()
-        return torch.nn.utils.clip_grad_norm_(parameters, max_norm, norm_type=norm_type)
+            if parse_flag_from_env("ACCELERATE_USE_FSDP", default=False):
```

Comment:
Please use `os.environ.get("ACCELERATE_USE_FSDP", "false") == "true"` as other uses in this file

Reply:
done